tabular/src/autogluon/tabular/models/vowpalwabbit/vowpalwabbit_model.py

```diff
@@ -163,9 +190,9 @@ def load(cls, path: str, reset_paths=True, verbose=True):
         if model._load_model:
             file_path = path + cls.model_internals_file_name
 
-            model_load_params = f" -i {file_path}"
+            model_load_params = f" -i {file_path} --quiet"
```

Comment:
Shouldn't this take verbosity into account?

Reply:
if quiet is removed it logs redundant information to what was logged during training, so it doesn't add value. It is also extremely verbose (more than we would like).

Reply:
Got it